.vsts/job.yml

```diff
@@ -61,14 +61,14 @@ jobs:
   #     verbose: false
   #     customCommand: 'run intTest'
 
-  - ${{ if eq(parameters.name, 'Windows') }}:
-    - task: Npm@1
-      condition: false
-      displayName: npm frameworkTest
-      inputs:
-        command: custom
-        verbose: false
-        customCommand: 'run frameworkTest'
+  #- ${{ if eq(parameters.name, 'Windows') }}:
+  - task: Npm@1
+    condition: false
```

Comment:
Shouldn't be remove this to have the tests run?

Reply:
Yep! :)